packages/components/src/components/ExternalCard/elements.ts

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import { Stack } from '../Stack';
+import { Text } from '../Text';
+
+export const StyledWrapper = styled(Stack)<{ thumbnail: string }>`
+  position: relative;
+  overflow: hidden;
+  height: 156px;
+  width: 276px;
+  background: #999999;
```

Comment:
Before the image loads the background is quite light, what do you think about changing it to `#161616` to be in line with the other cards? When the image fails to load this would also be a nice fallback to make sure the text is still readable.

Reply:
@tristandubbeld good catch, updated here: https://github.com/codesandbox/codesandbox-client/pull/7217/files#diff-c3fb0bcbf16750175cf4ed35ed5e3b66e6b0f026e36a21933f896816fba5e04bR11. I also added a `ThumblessVariant` showcasing the UI when the image is loading/fails to load.